tests/Aspire.Hosting.Seq.Tests/SeqFunctionalTests.cs

```diff
@@ -32,6 +32,9 @@ public async Task VerifySeqResource()
 
         var client = CreateClient(seqUrl);
 
+        // Wait for the Seq server to be fully initialized and healthy before ingestion
+        await WaitForHealthyAsync(client, default);
```

Comment:
Can we use `WaitForHealthyAsync` since you added a health check on the resource?

Reply:
Done.